app/pages/lab/workflow-viewer/answers.cjsx

```diff
@@ -0,0 +1,58 @@
+React = require 'react'
+Sty = require './jsplumb-style'
+icons = require '../../../classifier/tasks/drawing/icons'
+{Markdown} = require 'markdownz'
+
+# Styling for ansers within a list
+# AnswerItem takes in `input`, `eps`, and `jp` as props
+# `jp` is the jsPlumb instance for the visualizer
+# `eps` = {add: add endput callback, remove: remove endpoint callback, zIndex: the z-index of the task, endpionts: list of all endpoints}
+# `inputs` = {listId: unique uuid for endpoint, item: answer or tool, type: type of task, idx: current index of answer or tool}
+AnswerItem = React.createClass
+  displayName: 'AnswerItem'
+
+  # check if a string is just whitespace
+  isEmptyStr: (str) ->
+    if str.replace(/^\s+|\s+$/g, '').length == 0
+      return '&nbsp;'
+    else
+      return str
+
+  render: ->
+    if @props.inputs.type == 'drawing'
+      icon = <span className='drawing-tool-icon-vis' style={color: @props.inputs.item.color}>{icons[@props.inputs.item.type]}</span>
+    else
+      icon = undefined
+    <li className='answer-item' id={@props.inputs.listId}>
+      <div className='lab'>
+        {icon}
+        <Markdown className='markdown-vis'>{@isEmptyStr(@props.inputs.item.label)}</Markdown>
```

Comment:
A div with an empty line renders as a different height (nothing vs. single space).  Now that I think about it I can do the same thing with a min-height in the CSS.